File: Data Structures & Algorithms/anagram-groups/submission-2.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {

        map<string, vector<string>> mp;


        for(auto word : strs)
        {
            string w = word;

            sort(w.begin(), w.end());

            mp[w].push_back(word);
        }

        vector<vector<string>> res;

        for(auto v : mp)
        {
         res.push_back(v.second);
        }


        return res;

  
        
    }
};

File: Data Structures & Algorithms/length-of-last-word/submission-2.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:

        s = s.strip()

        word = ""

        if len(s) - 1 == 0:
            return 1

        for c in range(len(s)-1, 0, -1):
            if s[c] != " ":
                word = s[c] + word
            else:
                break

        return len(word)

        